index.html

```diff
@@ -643,24 +643,31 @@ <h2>Security</h2>
           </ul>
         </div>
         <p><span class="rfc2119-assertion" id="common-constraints-security-2">
-            Conformant Consumers MUST support all of these security schemes.</span>
+            Conformant Consumers MUST support at least all of these security schemes.</span>
         </p>
-	    
+
         <p><span class="rfc2119-assertion" id="common-constraints-security-3">
-            A Thing MAY implement multiple security schemes.</span>
+            A Thing MAY implement multiple security schemes, at least one of the above MUST be supported.</span>
         </p>
-        
+
+        <p>
+            The <a href="https://w3c.github.io/wot-thing-description/#basicsecurityscheme">
+                <code>BasicSecurityScheme</code>
+            </a> SHOULD be avoided if the underlying transport is not
+            <a href="https://w3c.github.io/wot-architecture/#sec-security-consideration-secure-transport">secure</a>.
```

Comment:
Whilst I don't disagree with this assertion (where it's possible to use TLS), I'm not sure this advice is unique to the Basic security scheme. A man-in-the-middle attack can be used to intercept tokens in the OAuth2 security scheme too.

Note that the [Security Considerations](https://w3c.github.io/wot-profile/#security-considerations) section already says:
> The security considerations of the [WoT Architecture](https://w3c.github.io/wot-architecture/#sec-security-consideration) and [WoT Thing Description](https://w3c.github.io/wot-thing-description/#sec-security-considerations) MUST be adopted by compliant implementations.

This includes WoT Architecture's [section](https://w3c.github.io/wot-architecture/#sec-security-consideration-secure-transport) on secure transport which you linked to, which goes into more detail and mentions that tokens are at risk as well as passwords.

Also note that assertions need to be wrapped in a `<div>` or a `<span>` with `class="rfc2119-assertion"` and a unique `id`.

Reply:
Some comments:
- We can change to MUST in the profile ?
- TLS should be for all security mechanisms indeed (except nosec).

Reply:
> We can change to MUST in the profile ?

Regular reminder: TLS doesn't work on local networks. This would effectively forbid the use of profiles on local networks.

Use case: WebThings Gateway currently exposes its API via http://gateway.local, and _optionally_ https://{subdomain}.webthings.io, and uses profiles on both.